File: flowcore/src/main/java/com/raizlabs/android/dbflow/data/Blob.java
package com.raizlabs.android.dbflow.data;

/**
 * Description: Provides a way to support blob format data.
 */
public class Blob {

    private byte[] blob;

    public Blob(byte[] blob) {
        this.blob = blob;
    }

    public void setBlob(byte[] blob) {
        this.blob = blob;
    }

    public byte[] getBlob() {
        return blob;
    }
}

File: dbflow-core-common/src/main/kotlin/com/raizlabs/dbflow5/data/Blob.kt
package com.raizlabs.dbflow5.data

import com.raizlabs.dbflow5.JvmOverloads

/**
 * Description: Provides a way to support blob format data.
 */
class Blob @JvmOverloads constructor(
    /**
     * Sets the underlying blob data.
     *
     * @param blob The set of bytes to use.
     */
    var blob: ByteArray? = null)
